### DISCUSSION TITLE
Linear integration

### DISCUSSION THREAD

**Discussion by xyfleet** [outline/outline#4427] [Ideas]
Hi Outline team,

Linear is becoming a popular product. (https://linear.app/)
Do you plan to integrate with Linear (a ticket system) soon?

**Comment by tommoor** [comment]
Definitely, any thoughts on the type of integration you'd like to see?

**Comment by xyfleet** [comment]
@tommoor Thanks. In general, we want to use Outline(Wiki) to trace the tickets created in linear and reports created in linear. 

(When we use JIRA and Confluence, we create an article in Confluence and use this one to trace the ticket's information like, status, priority and son in JIRA. We are also able to create report to collect the statistics of JIRA tickets. )

**Comment by trounceabout** [comment]
> Definitely, any thoughts on the type of integration you'd like to see?

+1 to this! Even a simple embed , something similar to Notion's Linear integration, would be fantastic 🙏

**Comment by problem** [comment]
For us to seriously consider Outline as a replacement to Confluence, we would need all of the basic behavior found in Confluence.

1. Pick up the issue ID automatically and unfurl it / turn it into an object where status and other metadata can be picked up
2. Display tables with the ability to pass in a query and filter on Linear issues
3. Ideally, deeper integration, like syncing in Projects from Linear, so that the hierarchy in Linear can be mirrored in Outline and you can create a repository of requirements docs in Outline without a lot of manual work

**Comment by xyfleet** [comment]
Any progress on this feature request? @tommoor Thanks.

**Reply by tommoor** [reply]
The work in https://github.com/outline/outline/pull/5308 is providing foundations for authenticated editor integrations

**Comment by wouter-deen** [comment]
Would love to see this!

**Comment by florian-obradovic** [comment]
Also would love to see this.

**Comment by arshtanov** [comment]
Any news on that? waiting for it

**Reply by hmacr** [reply]
This is available on cloud and v0.84.0.

**Reply by JosTheDude** [reply]
How does one set it up? Docs don't seem to have an update c:

**Reply by jjgarciarovira** [reply]
Same question over here, seeing: 

> The Linear Integration is currently disabled. Please set the associated environment variables and restart the server to enable integration

**Reply by tommoor** [reply]
Added here:
https://docs.getoutline.com/s/hosting/doc/linear-g8N2riMweL

**Reply by jjgarciarovira** [reply]
Thanks! Got it working!

**Reply by JosTheDude** [reply]
> Added here: https://docs.getoutline.com/s/hosting/doc/linear-g8N2riMweL

the link specified in the docs to make a new application is for the Outline Linear workspace btw -- should be a guide to how to navigate there instead

**Reply by tommoor** [reply]
👍 updated

**Reply by adriengibrat** [reply]
Task/issue mention are great and very helpful.

Any chance to see a deeper integration allowing to see kanban view. Or should i ask linear to add authorized domain for iframe domain/dedicated embedable URL without iframe restriction.

https://developer.mozilla.org/en-US/docs/Web/HTTP/Reference/Headers/X-Frame-Options#allow-from_origin

**Comment by tommoor** [comment]
It's compatible with both versions, anything making you think otherwise?
